frontend/src/bundles/common/components/header/components/notifications/components/notifications-bell.tsx

```diff
@@ -0,0 +1,48 @@
+import { BellIcon } from '@heroicons/react/24/solid';
+
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+
+type Properties = {
+    count: number;
+    onClick: () => void;
+    showList: boolean;
+};
+
+const NotificationBell = ({
+    count,
+    onClick,
+    showList,
+}: Properties): JSX.Element => {
+    const animate = count > 0 && !showList;
+
+    return (
+        <button
+            className={getValidClassNames(
+                'relative flex items-center justify-center',
+                animate && showList === false ? 'animate-pulse' : '',
+            )}
+            onClick={onClick}
+        >
+            <BellIcon
+                className={getValidClassNames(
+                    'text-lm-grey-200 w-8 transform transition-all duration-300 ease-in-out sm:w-6',
+                    showList === true ? 'rotate-12 scale-110' : '',
+                )}
```

Comment:
Why do we need `===`? Can we just leave it like this: `showList ? 'rotate-12 scale-110' : ''`  ?